scripts/develop.sh

```diff
@@ -17,18 +27,19 @@ echo '== Without these binaries, workspaces will fail to start!'
 # https://stackoverflow.com/questions/3004811/how-do-you-run-multiple-programs-in-parallel-from-a-bash-script
 (
 	cd "${PROJECT_ROOT}"
-
-	trap 'kill 0' SIGINT
+	# Send an interrupt signal to all processes in this subshell on exit
 	CODERV2_HOST=http://127.0.0.1:3000 INSPECT_XSTATE=true yarn --cwd=./site dev &
 	go run -tags embed cmd/coder/main.go server --in-memory --tunnel &
 
-	# Just a minor sleep to ensure the first user was created to make the member.
-	sleep 2
+	echo '== Waiting for Coder to become ready'
+	timeout 60s bash -c 'until nc -z 127.0.0.1 3000 > /dev/null 2>&1; do sleep 1; done'
```

Comment:
I suggest we use `curl` like in #2573. This ensures the API is giving actual HTTP responses.

Reply:
Good point; and `curl` is more likely to be present